server/data/incidentClient.ts

```diff
@@ -63,6 +63,17 @@ export default class IncidentClient {
     return results.rows[0]
   }
 
```

Comment:
get bookingId for offenderService.getOffenderDetails (see coordinator.ts L149) for displaying prisoner profile in success page. Previously the bookingId was obtained directly from the report with the call to reviewService.getReport(). However this only returns undeleted reports and at the moment the success screen is reached the current report will already have been deleted